pyproject.toml

```diff
@@ -27,6 +27,7 @@ watchtower = ">=1.0.0"
 boto3 = ">=1.16.13"
 botocore = ">=1.19.13"
 flask-caching = "*"
+flask-unleash = "^2.0.0"
```

Comment:
@upadhyeammit is inventory also using the same package? I am asking as it looks like t[he package is not maintained recently.](https://github.com/Unleash/Flask-Unleash)